src/whitenoise/compress.py

```diff
@@ -82,14 +82,14 @@ def compress(self, path):
             stat_result = os.fstat(f.fileno())
             data = f.read()
         size = len(data)
-        if self.use_brotli:
+        if self.use_brotli and not os.path.isfile(f"{path}.br"):
```

Comment:
Wouldn't this cause WhiteNoise to never re-compress a file, even if its contents have changed?

Reply:
Yes, exactly. And unfortunately, in the default config, the non-MD5-hashed names are left around and can still be served... This needs some more involved consideration, and tests.